ONSdigital/eq-author
# Digeq x angular prototype

**What**

Rewritten question builder using angular.js

**How to test**

1) Create a survey and a new questionnaire
2) Drag and drop elements from the 'Question Types' area into the main area on the left
3) Drag and drop elements from the main area to the bin
4) Complete the dragged forms and verify the debug json is poplated below
5) Save your work
6) Reload the page and verify your changes were saved

**Who can test**

Anybody except @weapdiv-david and @LJBabbage 

**Still outstanding**

Several CSS bugs which will be fixed when the latest UI changes are applied
